### PR TITLE
Change Find service name now that we are accepting undersgraduate courses

### PR DESCRIPTION
**Pull request by tomas-stefano**
### Context

We need to update the new service name for Find, so that we can enable Teacher Degree Apprenticeship (TDA) courses.

The new service name is:

Find teacher training courses

### Guidance to review

1. Check each page changed on this PR on the review app and the service name should match with the above.
2. Is there any other place mentioning postgraduate that we should change?

### What this PR does not do?

It doesn't change the links. This will be done in a future card/PR.

### CODE REVIEW

**Comment by tomas-stefano** [comment]
[Update the name of the Find service, and remove reference to postgraduate on Find and Apply](https://trello.com/c/ujqCUsFK/1593-update-the-name-of-the-find-service-and-remove-reference-to-postgraduate-on-find-and-apply)

**Review by avinhurry — COMMENTED** [review]
Are we updating all instances in this PR? There are quite a few places where we haven't updated it. E.G

- https://github.com/DFE-Digital/publish-teacher-training/blob/main/README.md
- https://github.com/DFE-Digital/publish-teacher-training/blob/b4ac7e7d4281259ef0f85540751d75e811036a21/config/initializers/canonical_rails.rb#L12
- https://github.com/DFE-Digital/publish-teacher-training/blob/b4ac7e7d4281259ef0f85540751d75e811036a21/config/initializers/content_security_policy.rb#L13

- [Do we need to speak to GIT about the referers](https://github.com/DFE-Digital/publish-teacher-training/blob/b4ac7e7d4281259ef0f85540751d75e811036a21/config/locales/find.yml#L194) 

- https://github.com/DFE-Digital/publish-teacher-training/blob/main/config/settings/production_aks.yml (and all of the other yml files)

**Comment by tomas-stefano** [comment]
> Are we updating all instances in this PR? There are quite a few places where we haven't updated it. E.G
> 
> * https://github.com/DFE-Digital/publish-teacher-training/blob/main/README.md
> * https://github.com/DFE-Digital/publish-teacher-training/blob/b4ac7e7d4281259ef0f85540751d75e811036a21/config/initializers/canonical_rails.rb#L12
> * https://github.com/DFE-Digital/publish-teacher-training/blob/b4ac7e7d4281259ef0f85540751d75e811036a21/config/initializers/content_security_policy.rb#L13
> * [Do we need to speak to GIT about the referers](https://github.com/DFE-Digital/publish-teacher-training/blob/b4ac7e7d4281259ef0f85540751d75e811036a21/config/locales/find.yml#L194)
> * https://github.com/DFE-Digital/publish-teacher-training/blob/main/config/settings/production_aks.yml (and all of the other yml files)

This PR is just the content changes. The domain/url changes will be done in a separate PR. Yes we do need to talk with GiT yes :)

**Review by avinhurry — APPROVED** [review]
Looks good to me then 🚢

**Review by dcyoung-dev — APPROVED** [review]

File: app/views/pages/terms.html.erb [hunk 1]
```diff
@@ -115,7 +115,7 @@
     </p>
     <ul class="govuk-list govuk-list--bullet">
       <li>
-        listings for postgraduate teacher training courses that aren’t relevant to the service, such as but not limited to non-teacher training courses, undergraduate courses, non-primary and secondary courses and non-English courses
+        listings for teacher training courses that aren't relevant to the service, such as but not limited to non-teacher training courses, non-primary and secondary courses and non-English courses
```
Comment (avinhurry) [inline app/views/pages/terms.html.erb:118]: Smart quotes please 👀

Comment (tomas-stefano) [reply app/views/pages/terms.html.erb]: Good spot. Done!

**Pull request merged**